sirepo/package_data/static/js/raydata.js

```diff
@@ -575,14 +575,17 @@ SIREPO.app.directive('scanSelector', function() {
                 <table class="table table-striped table-hover col-sm-4">
                   <thead>
                     <tr>
-                      <th data-ng-repeat="column in columnHeaders track by $index" data-ng-mouseover="hoverChange($index, true)" data-ng-mouseleave="hoverChange($index, false)" style="width: 100px; height: 40px;">
+                      <th data-ng-repeat="column in columnHeaders track by $index" data-ng-mouseover="hoverChange($index, true)" data-ng-mouseleave="hoverChange($index, false)" data-ng-click="sortCol(column)" style="width: 100px; height: 40px;">
+                        <span style="color:lightgray;" class="glyphicon glyphicon-arrow-down" data-ng-show="showArrowDown(column)" ></span>
+                        <span style="color:lightgray" class="glyphicon glyphicon-arrow-up" data-ng-show="showArrowUp(column)" ></span>
                         {{ column }}
+                        <input type="checkbox" data-ng-checked="selectAllColumns" data-ng-show="showSelectAllButton($index)" data-ng-click="toggleSelectAll()"/>
                         <button type="submit" class="btn btn-primary btn-xs"  id="del-column" data-ng-show="showDeleteButton($index)" data-ng-click="deleteCol(column)"><span class="glyphicon glyphicon-remove"></span></button>
                       </th>
                   </tr>
                   </thead>
-                  <tbody ng-repeat="s in scans">
-                    <tr>
+                  <tbody>
+                    <tr ng-repeat="s in scans|orderBy:orderByColumn:reverseSortScans">
```

Comment:
Nice! One spacing nit `s in scans | orderBy:orderByColumn:reverseSortScans`

Reply:
eb41875ad13fdf2a9ff2f5e261bcf5432c048ace